packages/react/src/animation/Animation.features.stories.tsx

```diff
@@ -1,256 +1,84 @@
-                  fillOpacity: 1,
-                  fillRule: 'evenodd',
-                  stroke: 'none',
-                }}
-                transform="matrix(1.01883 0 0 -1.01883 -29.952 104.07)"
-              />
-              <path
-                d="M188.706 51.805c.148.868.37 2.533.37 4.415 0 8.736-4.435 17.604-16.11 17.604-12.488 0-18.252-9.877-18.252-18.835 0-11.08 7.02-18 19.287-18 4.877 0 9.384.72 13.08 2.234l-1.48 7.418c-3.029-1.005-6.132-1.507-9.974-1.507-5.247 0-9.828 2.153-10.198 6.743l23.277-.07zm-23.35 7.599c.295 2.884 2.216 7.138 7.02 7.138 5.097 0 6.28-4.543 6.28-7.138h-13.3zM216.801 86.772V69.788h-.147c-1.627 2.377-5.026 3.963-9.533 3.963-8.646 0-16.257-6.92-16.184-18.741 0-10.958 6.873-18.094 15.444-18.094 4.656 0 9.09 2.019 11.306 5.912h.222l.444-5.119h9.976c-.148 2.379-.297 6.488-.297 10.524v38.539h-11.231zm0-33.421c0-.864-.074-1.73-.222-2.45-.664-3.1-3.325-5.262-6.576-5.262-4.656 0-7.685 3.748-7.685 9.659 0 5.55 2.586 10.019 7.758 10.019 3.473 0 5.912-2.378 6.578-5.333.147-.65.147-1.371.147-2.019v-4.614z"
-                style={{
-                  fill: 'inherit',
-                  fillOpacity: 1,
-                  fillRule: 'evenodd',
-                  stroke: 'none',
-                }}
-                transform="matrix(1.01883 0 0 -1.01883 -29.952 104.07)"
-              />
-              <path
-                d="M261.873 37.749v33.794H250.64V37.749h11.233zm-5.617 38.408c3.917 0 6.355 2.595 6.355 5.838-.073 3.316-2.438 5.839-6.28 5.839-3.843 0-6.355-2.523-6.355-5.839 0-3.243 2.438-5.838 6.207-5.838h.073zM268.088 37.749h11.232v18.872c0 1.01.073 2.019.37 2.741.812 2.018 2.66 4.108 5.763 4.108 4.065 0 5.69-3.099 5.69-7.642V37.749h11.232v19.377c0 10.38-5.541 15.21-12.931 15.21-6.06 0-8.72-3.387-10.198-5.694h.075v4.901h-11.233c.148-3.171 0-33.794 0-33.794z"
-                style={{
-                  fill: 'var(--brand-color-canvas-default)',
-                  fillOpacity: 1,
-                  fillRule: 'evenodd',
-                  stroke: 'none',
-                }}
-                transform="matrix(1.01883 0 0 -1.01883 -29.952 104.07)"
-              />
-            </svg>
-          </Animate>
-        </LogoSuite.Logobar>
-      </LogoSuite>
-    </AnimationProvider>
-  )
+IntersectionObserver.argTypes = {
+  visibilityOptions: {
+    options: ['bottom-of-screen', 'middle-of-screen', 'about-to-leave'],
+    control: 'radio',
+  },
+  variant: {
+    name: 'animate',
+    options: AnimationVariants,
+    control: 'select',
+    table: {
+      category: 'Children',
+    },
+  },
 }
 
-export const AnimationOnPress = () => {
-  return (
-    <AnimationProvider animationTrigger="click">
-      <Stack className={styles.Playground} justifyContent="center" alignItems="center" animate="fade-out">
-        <Text as="p" className={styles.PlaygroundText}>
-          Fade out on press
-        </Text>
-      </Stack>
-    </AnimationProvider>
-  )
+export const AdvancedAnimation = args => (
```

Comment:
Is advanced referring to the use of multiple parameters like delay, duration etc at the same time? I feel like this example is quite similar to Playground or might be better served in our docs tbh.